apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -12,56 +13,76 @@ import {
   CoinSelectionRecipient,
   CoinSelectionUtxo,
   getBitcoinFees,
+  isValidBitcoinTransaction,
   payerToBip32Derivation,
 } from '@leather.io/bitcoin';
-import { AverageBitcoinFeeRates } from '@leather.io/models';
+import {
+  AverageBitcoinFeeRates,
+  Money,
+  bitcoinNetworkToNetworkMode,
+  createBitcoinAddress,
+} from '@leather.io/models';
 import { createMoneyFromDecimal } from '@leather.io/utils';
 
 import {
   CreateCurrentSendRoute,
   createCoinSelectionUtxos,
-  formatBitcoinError,
   useSendSheetNavigation,
   useSendSheetRoute,
 } from '../../send-form.utils';
 import { SendFormBtcContext } from '../providers/send-form-btc-provider';
 import { SendFormBtcSchema } from '../schemas/send-form-btc.schema';
+import { formatBitcoinError } from '../validation/btc.validation';
 
 type CurrentRoute = CreateCurrentSendRoute<'send-form-btc'>;
 
-function parseSendFormValues(values: SendFormBtcSchema) {
+interface ParsedSendFormValues {
+  amount: Money;
+  recipients: CoinSelectionRecipient[];
+}
+
+function parseSendFormValues({
+  amount: inputAmount,
+  recipient,
+}: SendFormBtcSchema): ParsedSendFormValues {
+  const amount = createMoneyFromDecimal(new BigNumber(inputAmount), 'BTC');
   return {
-    amount: createMoneyFromDecimal(new BigNumber(values.amount), 'BTC'),
+    amount,
     recipients: [
       {
-        address: values.recipient,
-        amount: createMoneyFromDecimal(new BigNumber(values.amount), 'BTC'),
+        address: createBitcoinAddress(recipient),
```

Comment:
Here I create a type `BitcoinAddress` using the raw form value but it could be better to add that validation in `SendFormBtcSchema`.

We have more UI changes to make to the forms though so am trying to impact those as little as I can. 

I think it's nicer for users to have the validation triggered on the `Review` button then after entering the value as we show a `Toast`